app/views/widgets/show.html.erb

```diff
@@ -37,24 +37,40 @@
       <% elsif @existing_track %>
         <%= link_to update_url(:track_id => @existing_track.id, :track_medium => "delete", :r => show_user_profile_url(:url_name => @user.url_name)), :target => '_top',
           :class => 'alaveteli-widget__bottom__action alaveteli-widget__button--unsubscribe',
-          :onclick => "if (pageTracker) { pageTracker._trackEvent('widgets', 'unsubscribe', window.top.location.href, 1)};" do %>
+          :onclick => "#{track_analytics_event(
```

Comment:
Not sure why the helper calls in this file are interpolated where others aren't.

Can also drop the braces on the options hash.